chainer/links/model/vision/resnet.py

```diff
@@ -154,8 +154,7 @@ def __call__(self, x, layers=['prob']):
                 target_layers.remove(key)
         return activations
 
-    def extract(self, images, layers=['pool5'], size=(224, 224),
-                volatile=flag.OFF):
+    def extract(self, images, layers=['pool5'], size=(224, 224)):
```

Comment:
This change removes the flag that enables/disables backprop? Are you thinking of letting users switch it outside of this function?